events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
-# Communications Lead Handbook
+# Communications Team Handbook
 
 This document defines communications activities needed to run a
-Contributor Summit.
+Kubernetes Contributor Summit (KCS). The Comms Team consists of 1-2 lead(s), shadows, volunteers, at least 1 emeritus lead, and a Website Manager.
 
-- [Overview](#overview)
-- [Skills and Qualifications](#skills-and-qualifications)
-- [Responsibilities](#responsibilities)
-  - [Communication Channels and Property Access](#communication-channels-and-property-access)
+Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.
+
+Shadows will learn both how to do comms and how to run the KCS Comms team, as directed by the Comms Lead. They are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). Must either have [Kubernetes Org Membership] or be planning to pursue org membership.
+
+Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].
+
+An emeritus comms lead is a past lead who provides mentorship to the current comms lead. Should have [Kubernetes Org Membership].
```

Comment:
Should to Must

If someone is an emeritus comms lead, I would hope that we'd grant org membership as soon as we could. But, what examples were you thinking of here?

Reply:
So the requirements below says the "KCS Comms Lead must have \[Kubernetes Org Membership\]. (They should not plan to pursue membership during the cycle.)"  
  
Then we definitely update this language for org membership as a requirement.

Reply:
I guess we could have the case, someone lost the membership because of inactivity and came back and picked up by supporting. But i don't know if this ever happened

Reply:
I was thinking of a case like Mario is describing here. Like I don't know if Paris' org membership is still active, but imagine it wasn't and we needed help and she volunteered to serve as an emeritus advisor. I'd still accept that. Unlikely. But possible.